phoenix-core/src/main/java/org/apache/phoenix/execute/AggregatePlan.java

```diff
@@ -51,6 +51,7 @@
 import org.apache.phoenix.iterate.SequenceResultIterator;
 import org.apache.phoenix.iterate.SpoolingResultIterator;
 import org.apache.phoenix.iterate.UngroupedAggregatingResultIterator;
+import org.apache.phoenix.jdbc.PhoenixStatement.Operation;
```

Comment:
Is there no other change other than this import in the file? Probably should be reverted then?